java/CMakeLists.txt

```diff
@@ -128,6 +128,7 @@ set(JAVA_MAIN_CLASSES
   src/main/java/org/rocksdb/ComparatorType.java
   src/main/java/org/rocksdb/CompressionOptions.java
   src/main/java/org/rocksdb/CompressionType.java
+  src/main/java/org/rocksdb/ConfigOptions.java
   src/main/java/org/rocksdb/DataBlockIndexType.java
```

Comment:
You also need to add  `java/rocksjni/config_options.cc` to `java/CMakeLists.txt`